src/generated/api/eventsApi.ts

```diff
@@ -112,11 +112,13 @@ export class EventsApi {
   }
 
   /**
-   * Send an event to the ranking system after a user interacts with a search result.  When querying a collection, you can set the tracking type of the query request. When it is `CLICK` or `POS_NEG`, a token is generated for each result in the query response. You can use this token to provide feedback to the ranking system. Each time you want to record an event on a particular search result, use the send event call and provide:  - The `name` of the event, e.g. `click`, `purchase`. - The `token` from the search result. - The `weight` to assign to the event, e.g. `1`. - An object containing any additional `metadata`.  For example, to send an event where a customer purchased a product, use the following call:  ```json {   \"name\": \"purchase\",   \"token\": \"eyJ...\",   \"weight\": 1,   \"metadata\": {     \"discount\": 0.2,     \"margin\": 30.0,     \"customer_id\": \"12345\",     \"ui_test_segment\": \"A\"   } } ```
+   * Send an event to the ranking system after a user interacts with a search result.  When querying a collection, you can set the tracking type of the query request. When it is `CLICK` or `POS_NEG`, a token is generated for each result in the query response. You can use this token to provide feedback to the ranking system. Each time you want to record an event on a particular search result, use the send event call and provide:  - The `name` of the event, e.g. `click`, `purchase`. - The `token` from the search result. - The `weight` to assign to the event, e.g. `1`. - An object containing any additional `metadata`.  For example, to send an event where a customer purchased a product, use the following call:  ```json {   \"name\": \"purchase\",   \"token\": \"eyJ...\",   \"weight\": 1,   \"metadata\": {     \"discount\": 0.2,     \"margin\": 30.0,     \"customer_id\": \"12345\",     \"ui_test_segment\": \"A\"   } } ```  Note: Unlike other API calls, the `SendEvent` call requires the `Account-Id` header to be set to your account ID. This is because other API calls infer the account ID from your API key.
    * @summary Send event
+   * @param accountId Unlike other API calls, the &#x60;SendEvent&#x60; call requires the &#x60;Account-Id&#x60; header to be set to your account ID. This is because other API calls infer the account ID from your API key.
    * @param sendEventRequest
    */
   public async sendEvent(
+    accountId: string,
```

Comment:
Unfortunately this is a major version bump:

> MAJOR version when you make incompatible API changes,

https://semver.org/

That's OK though, this is our reality. We just go to 6.0.0.

Reply:
Ah if we're at v4 now, then yep v5. I thought we were on v5 already but I must have misread.

And yep, if the account ID field is required for an RPC, we should pass accountId as the first argument in those cases. If it's optional, I suppose we're going to need to allow them to pass it another way?